support-frontend/assets/components/productPage/productPageHero/productPageHero.jsx

```diff
@@ -29,9 +27,17 @@ type PropTypes = {|
   heading: string,
   content?: Option<Node>,
   hasCampaign: boolean,
-  dailyEditionsVariant?: boolean
+  dailyEditionsVariant?: boolean,
```

Comment:
I don't think this prop is needed any more, it doesn't seem to be used in this component.